Update CCC_TAC_Minutes-2022-04-07.md (#102)
confidential-computing/governance: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md

# Confidential Computing Consortium 

> ##Minutes of the Technical Advisory Council 

Author: Formatting for sub-heading broken, I will fix in next revision

* **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io) ([ical](https://calendar.google.com/calendar/ical/c_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics), [Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com))
* **Recordings**: [YouTube Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)

### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* **Accenture** - Giuseppe Giordano
* **ANT Group** - Zongmin Gu

> * **ARM** - Thomas Fossati / Michael Lu

Reviewer: Revert.  The correct capitalization is "Arm".  See https://www.arm.com/company/policies/trademarks

And according to https://en.wikipedia.org/wiki/Ant_Group "Ant Group" was also correct as it was.

### Welcome

Rep. Thaler opened the call at 7:04am PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.

### Roll Call / Attendance

#### Voting members

* Simon Johnson (Intel)
* Dave Thaler (Microsoft)
* Lily Sturman (Red Hat)

> * Thomas Fossati (ARM)

Reviewer: ```suggestion
* Thomas Fossati (Arm)
```

(Line 69 on the other hand is correct)

* Eric Northup (Meta)

#### Attendees

* Eric Voit (Cisco)
* Nick Vidal (Profian)
* Penglin Yang (China Mobile)
* Steve Van Lare (Anjuna)
* Ben Fischer (Red Hat)
* Mark Novak (JP Morgan Chase)
* Naveen Cherukuri (NVIDIA)
* Jethro Beekman (Fortanix)
* Henk Birkholz (Fraunhofer-Gesellschaft)
* Beni Eugster (Swisscom)
* Ionut Mihalcea (Arm)
* Vijay Nayani (Huawei)
* Xinxin Fan (IoTeX)
* Marc Meunier (Arm)
* Sampo Sovio (Huawei)

#### Also Participating
* Kurt Taylor (Linux Foundation)
* Helen Lau (Linux Foundation)        

### Meeting minute approvals

#### Approved by voting members

> **Resolved:** That the minutes of the [March 10, 2022](../2022-03-10/TAC_Minutes-2022-03-10.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.

Reviewer 2: s/pdf/md
https://github.com/confidential-computing/governance/blob/main/TAC/Meetings/2022/2022-03-10/CCC_TAC_Minutes-2022-03-10.md

Reviewer: ```suggestion
**Resolved:** That the minutes of the [March 10, 2022](../2022-03-10/TAC_Minutes-2022-03-10.md) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```

 * Quorum reached, Minutes of March 10, 2022 were approved

#### Pending approval
 * None

### TAC Tech Talks
 *  Mark Novak - CC Governance Framework
 	*  Mark N, Thomas F, Naveen C, Steve VL, and Eric N expressed interest in collaborating on the governance topic, SIG formation possible

>  *  Eric Voit, Penglin Yang - Networking Confidential Computing

Reviewer: Please update minutes with links to the slide decks

Author: The slides and agenda are in the same meeting folder as the minutes

Reviewer: Thats why it should be trivial to link them.
```suggestion
 *  Mark Novak - [Confidential Computing Governance](Confidential%20Computing%20Governance.pdf)
 	*  Mark N, Thomas F, Naveen C, Steve VL, and Eric N expressed interest in collaborating on the governance topic, SIG formation possible
 *  Eric Voit, Penglin Yang - [Networking Confidential Computing](CCC-Networking-Tech-Talk-7Apr2022.pdf)
```

### Open Action Items

> 1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.

Reviewer: Revert.  We have two Eric's (Eric N and Eric V) that are regular attendees so need to make sure the minutes accurately reflect which one in minutes.  (Line 103 still has the V so is fine)

1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.

>     * Dan Middleton has opened a PR for review

Reviewer 2: ```suggestion
    * Dan Middleton has opened a PR for review https://github.com/confidential-computing/governance/pull/92
```

> 1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.

Reviewer 2: duplicate of line 98

Reviewer: I believe the list of names in in red above is correct (Mike and Eric did their action items)

> 1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.

Author: ```suggestion
1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

    * Open Enclave maintainers are close to 100% trained (Dave Thaler)
    * Veracruz lead has taken the course (Thomas Fossati).
    * Please report back that project maintainers have been made aware of the training, and have been asked to take it.

> 1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (DT) summarized options for resolution: 1) Adopt new stages 2) talk about proposal and fix it 3) delete the proposed new terms and just use the the two currently adopted stages, sandbox and incubation

Reviewer: ```suggestion
1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (DT) summarized options for resolution: 1) Adopt new stages 2) talk about proposal and fix it 3) delete the proposed new terms and just use the two currently adopted stages, sandbox and incubation
```

### Completed Action Items

> 1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for March 24, 2022.

Reviewer: ```suggestion
1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for April 21, 2022.
```

> 1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.

Reviewer: ```suggestion
1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.  (Done: https://github.com/confidential-computing/governance/issues/94)
```

1. **[Brian Warner]** Update the Google Sheet containing the [list of CCC dependencies](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0) with the changed dependencies noted in the project review document.

> 1. **[Brian Warner]** Add Enarx annual review to the agenda for March 10th TAC meeting.

Reviewer 2: Is this maybe a copy paste from an earlier minutes?

1. **[Brian Warner]** Add slide to this deck reminding the TAC to review PRs and issues, time permitting.

> 1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 19, 2022.

Reviewer 2: s/19/21
```suggestion
1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 21, 2022.
```

    * Confirmed, Shubhra Kar and David Deal from LF to present.

### Project overviews

### Interns
* Nick Vidal (NV) update on Interns, slack #general discussion, help and guidance 

> * Deferr to next meeting when Dan Middleton will be participating 

Reviewer: ```suggestion
* Defer to next meeting when Dan Middleton will be participating 
```

### Outreach update

* Ben Fischer (BF) proposed May webinar panel talk for The Future of Attestation
	* Asking for moderator and participants
	* (DT) mentioned the list of past TAC talks as possible outreach webinar topics, Mark N, Simon J, Dave T, Henk, Thomas F
	* (BF) to organize
* (DT) discussed needed updates to the Trusted Computing wikipedia page, also create a CCC or Confidential Computing page (none currently exists)
	* (NV) suggested the interns do an initial wikipedia page for CC, (NV) will work with them 
	* (NV) authoring a blog, will announce soon	 

### Common terminology
 * (DT) discussed Google Doc for terminology
 * Eric Voit (EV) raised multi node question as previously discussed with regard to confidential cloud, multi node is a hard problem to define, will defer to a future version
 * Steve Van Lare (SVL) will respond to comments in document

>  * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of CC, (at rest, in motion, and in use) 

Reviewer: ```suggestion
 * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of data protection (at rest, in motion, and in use) 
```

>  * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, I'll find out creative services timeline

Reviewer: ```suggestion
 * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, KT to find out creative services timeline
```
Also this means KT has an action item that should be listed in the action item list
